Source/Scene/GltfLoader.js

```diff
@@ -698,24 +698,20 @@ function loadAccessor(loader, gltf, accessorId, useQuaternion) {
   const bufferViewId = accessor.bufferView;
   if (defined(bufferViewId)) {
     const bufferViewLoader = loadBufferView(loader, gltf, bufferViewId);
-    const promise = bufferViewLoader.promise
-      .then(function (bufferViewLoader) {
-        if (loader.isDestroyed()) {
-          return;
-        }
-        const bufferViewTypedArray = bufferViewLoader.typedArray;
-        const packedTypedArray = getPackedTypedArray(
-          gltf,
-          accessor,
-          bufferViewTypedArray
-        );
+    const promise = bufferViewLoader.promise.then(function (bufferViewLoader) {
+      if (loader.isDestroyed()) {
+        return;
+      }
+      const bufferViewTypedArray = bufferViewLoader.typedArray;
+      const typedArray = getPackedTypedArray(
+        gltf,
+        accessor,
+        bufferViewTypedArray
+      );
 
-        useQuaternion = defaultValue(useQuaternion, false);
-        loadAccessorValues(accessor, packedTypedArray, values, useQuaternion);
-      })
-      .catch(function () {
-        loadDefaultAccessorValues(accessor, values);
-      });
```

Comment:
It seemed unsafe to catch and ignore errors from the buffer view loader so I removed it. The unit tests still pass so maybe this wasn't actually needed?